src/odemis/acq/stream/_projection.py

```diff
@@ -713,13 +714,20 @@ def _project2RGBPolar(self, ebeam_pos, pol_pos, cache_raw):
                 else:
                     colormap = "seismic"
 
+                # data including normalization need cropping of the pole, to ensure proper scaling
+                if pol_pos in [MD_POL_UP, MD_POL_DOP, MD_POL_DOCP, MD_POL_DOLP]:  # TODO: hardcoded for now
+                    hole = True
+                else:
+                    hole = False
```

Comment:
Technically, you could just write `hole = pol_pos in ...`, but it might be a little harder to read. So I let it up to you.